onnx2torch/node_converters/eye.py

```diff
@@ -0,0 +1,50 @@
+__all__ = [
+    'OnnxEyeLike',
+]
+
+import torch
+from torch import nn
+
+from onnx2torch.node_converters.registry import add_converter
+from onnx2torch.onnx_graph import OnnxGraph
+from onnx2torch.onnx_node import OnnxNode
+from onnx2torch.utils.common import OnnxToTorchModule
+from onnx2torch.utils.common import OperationConverterResult
+from onnx2torch.utils.common import onnx_mapping_from_node
+
+
+class OnnxEyeLike(nn.Module, OnnxToTorchModule):  # pylint: disable=missing-docstring
+    def __init__(self, eyelike_k: int, dtype: torch.dtype = None):
+        super().__init__()
+        self.dtype = dtype
+        self.eyelike_k = eyelike_k
+
+    def forward(self, x: torch.Tensor) -> torch.Tensor:  # pylint: disable=missing-function-docstring
+        if len(x.shape) != 2:
+            raise ValueError('OnnxEyeLike only support 2-D tensor')
+
+        input_value_n = x.size(dim=0)
+        input_value_m = x.size(dim=1)
+        if self.eyelike_k > input_value_n:
+            raise ValueError(
+                f'Error EyeLike Attribute k value, the k value is {self.eyelike_k},'
+                'but x shape is {(input_value_n, input_value_m)}'
```

Comment:
```suggestion
                f'but x shape is {(input_value_n, input_value_m)}'
```

Reply:
It is not clear what error occurs. Could you rewrite description to something like: "k should be greater than ..."?